QHotkey/qhotkey_x11.cpp

```diff
@@ -1,7 +1,21 @@
 #include "qhotkey.h"
 #include "qhotkey_p.h"
-#include <QDebug>
-#include <QX11Info>
+
+#if QT_VERSION >= QT_VERSION_CHECK(6, 2, 0)
+	#include <QGuiApplication>
+	
+	#define _NATIVE_EVENT_RESULT qintptr
+	#define _OVERRIDE_IDENTIFIER override
+#elif QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)
+	#error "Qt 6.2.0 or greater is required when using Qt6"
+#else
+	#include <QDebug>
+	#include <QX11Info>
+	
+	#define _NATIVE_EVENT_RESULT long
```

Comment:
Maybe use `QHOTKEY_NATIVE_EVENT_RESULT` just for consistency with `QHOTKEY_HASH_SEED`?

```suggestion
	#define QHOTKEY_NATIVE_EVENT_RESULT long
```

Reply:
I prefixed the definitions with `QHOTKEY_` only on the ones that are in the public header `QHotkey/qhotkey.h`, as it will go to the public for use. For a definition that is used only locally/internally on a .cpp file, I let it with just an underscore.

I think that making this differentiation in important, because we can quickly know what type of definition it is (if public or merely local/internal) by just looking at the prefix.

But as you wish :)

Please confirm if you want me to push this change.

Reply:
You convinced me, let's leave it as is.